tests/test_share_convert.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+
+import numpy as np
+import tensorflow as tf
+import tf_encrypted as tfe
+from tf_encrypted.tensor import int64factory
+
+
+class TestShareConvert(unittest.TestCase):
+
+    def setUp(self):
+        self.config = tfe.LocalConfig([
+            'server0',
+            'server1',
+            'crypto_producer'
+        ])
+
+    def test_share_convert(self):
+
+        prot = tfe.protocol.SecureNN(
+            tensor_factory=int64factory
+        )
+
+        val_a = np.array([100, -100])
```

Comment:
could we try with a bunch of random values instead? i'm worried this won't catch corner cases